src/proto/common.proto

```diff
@@ -0,0 +1,82 @@
+// API for the client to interact with the coordinator.
+
+syntax = "proto3";
+
+package common;
+
+import "google/protobuf/timestamp.proto";
+
+// Identifies which cohort is responsible for executing the operation.
+message Namespace {
+  string identifier = 1;
+}
+
+// Using a message to allow for extending to support multiple keys (e.g.
+// Get(StartsWith("foo"))).
+message Get {
+  // Scoped within the current namespace.
+  string key = 1;
+}
+
+message ConstantValue {
+  oneof type {
+    int64 int64_value = 1;
```

Comment:
NOTE: We can even stick with one of these for now.

Reply:
SGTM I changed it to be only int64 for now.